cloudbuild.yaml

```diff
@@ -20,8 +20,24 @@
 steps:
 
 - name: 'docker/compose'
-  id: 'Launch HAPI Source and Sink Servers'
-  args: ['-f', './docker/hapi-compose.yml', '-f', './docker/sink-compose.yml', 'up', '--force-recreate', '--remove-orphan', '-d']
+  id: 'Launch HAPI Source Server'
+  args: ['-f', './docker/hapi-compose.yml', '-p', 'hapi-compose', 'up', '--force-recreate', '--remove-orphan', '-d']
+
+- name: 'docker/compose'
+  id: 'Launch Sink Server 1'
+  args: ['-f', './docker/sink-compose.yml', '-p', 'sink-server1', 'up', '--force-recreate', '--remove-orphan', '-d' ]
```

Comment:
nit: instead of `sink-server1` and `sink-server2` maybe we can use better names, e.g., `sink-server-search`, `sink-server-jdbc`.

Reply:
Makes sense. Done